app/src/main/java/com/capstone/weatherapp/WeatherCache.kt

```diff
@@ -0,0 +1,72 @@
+package com.capstone.weatherapp
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class CityCache(
+    @PrimaryKey val id: Int,
+    @ColumnInfo(name = "name") val name: String,
+    val country: String,
+    val temp: Double,
+    val temp_min: Double,
+    val temp_max: Double,
+    val humidity: Double
+) {
+    fun convertToWeatherApiResponse(cityCacheList: List<CityCache>) : WeatherApiResponse {
+        val cityList = mutableListOf<City>()
+        for (cityCache: CityCache in cityCacheList) {
+            val city = City(
+                id = cityCache.id,
+                name = cityCache.name,
+                coord = Coord(0.0, 0.0),
```

Comment:
let's make all these zero's constants in a constants.kt file since we'll be using them in multiple files